riscv/tests/riscv.rs

```diff
@@ -615,12 +615,12 @@ fn many_chunks_memory() {
 }
 
 fn verify_riscv_crate(case: &str, inputs: &[u64], executor_witgen: bool) {
-    verify_riscv_crate_bb(case, inputs.iter().map(|&x| x.into()).collect());
     verify_riscv_crate_gl(
         case,
         inputs.iter().map(|&x| x.into()).collect(),
         executor_witgen,
     );
+    verify_riscv_crate_bb(case, inputs.iter().map(|&x| x.into()).collect());
```

Comment:
some tests rely on running GL before BB

Reply:
I think it was a test that expects a fail instruction from the executor, but since the executor isn't implemented for BB it failed differently